docs/how-to.md

```diff
@@ -259,12 +259,30 @@ The certificate must be signed by a chain of certificates which must be already
   
   Example: `dclcli tx pki add-x509-cert --certificate="----BEGIN CERTIFICATE----- ......" --from=jack`  
     
-##### 2. Add a new model info with the given VID/PID
 
 
-Command: `dclcli tx model add-model --vid=<uint16> --pid=<uint16> --productName=<string> --productLabel=<string or path> --sku=<string> 
---softwareVersion=<uint32> --softwareVersionString=<string> --hardwareVersion=<uint32> --hardwareVersionString=<string> --cdVersionNumber=<uint16> 
---from=<account>`
+##### 2. Add Vendor Info
+
+Add a new vendor contact info.
+
+  Role: `Vendor`
+  
+  Command: `dclcli tx vendorinfo add-vendor --companyLegalName=<string> --companyPreferredName=<string> --vendorName=<string> --vid=<uint16> ----vendorLandingPageURL=<url> --from=<account>`
+
+  Flags:
+  - companyLegalName `string (max64)` - Company Legal Name
+  - companyPreferredName `optional string (max64)` - Company Preferred Name  
+  - vendorLandingPageURL `optional string (max256)` - Landing Page URL for the Vendor
+  - vendorName `string (max32)` - Vendor Name
+  - vid `uint16` - Vendor ID      
+  
+
+  Example: `dclcli tx vendorinfo add-vendor --companyLegalName="XYZ Technology Solutions" --vid=123 --vendorName="XYZ Inc" --from="jack"`
+##### 3. Add a new model info with the given VID/PID
+
+  Role: `Vendor`
+
+  Command: `dclcli tx model add-model --vid=<uint16> --pid=<uint16> --productName=<string> --productLabel=<string or path> --sku=<string> --softwareVersion=<uint32> --softwareVersionString=<string> --hardwareVersion=<uint32> --hardwareVersionString=<string> --cdVersionNumber=<uint16> --from=<account>`
 
```

Comment:
The command template and flag below looks outdated (sku fields for example).
Actually I think it should be fine to not list all flags with all description here (remove them for all commands), and reference to `cli-help.md` instead.
I think just a command template as on line 285 (but with all available fields) and an example is enough.